pkgs/tools/networking/isync/default.nix

```diff
@@ -6,22 +6,17 @@
 , withCyrusSaslXoauth2 ? false, cyrus-sasl-xoauth2, makeWrapper
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "isync";
-  version = "1.4.4";
+  version = "1.5.0";
 
   src = fetchurl {
-    url = "mirror://sourceforge/isync/${pname}-${version}.tar.gz";
-    sha256 = "1zq0wwvmqsl9y71546dr0aygzn9gjjfiw19hlcq87s929y4p6ckw";
+    url = "mirror://sourceforge/isync/isync-${finalAttrs.version}.tar.gz";
+    sha256 = "oMgeEJOHvyedoWFFMQM5nneUav7PXFH5QTxedzVX940=";
```

Comment:
Please change this to `hash = "sha256-11111111762490922d0fe8a16ec288c7cba97abbaf8=";` (or whatever the proper SRI hash is).